modules/ShipStation/Api/src/ShipStationBundle.php

```diff
@@ -54,7 +58,7 @@ public static function getMajorVersion()
      */
     public static function getMinorVersion() 
     {
-        return '1.2';
+        return '2';
```

Comment:
It looks like you can actually just remove `getMajorVersion` and `getMinorVersion` from this class. It's all in `main.yaml` now: https://developer.x-cart.com/5.4/misc/x-cart_versions#x-cart-54x-versioning